python/mxnet/contrib/quantization.py

```diff
@@ -109,6 +110,9 @@ def _quantize_symbol(sym, ctx, excluded_symbols=None, excluded_operators=None,
         The quantized destination type for input data.
     quantize_mode: str
         The mode that quantization pass to apply.
+    quantize_granularity: str
+        The granularity of quantization, currently supports 'tensor-wise' and 'channel-wise'
```

Comment:
how about simplifying to `tensor` and `channel`?

Reply:
I thought `tensor` and `channel` is not that descriptive for the argument. There's also `per-axis`/`per-channel` and `per-tensor` naming, which is similar to `channel-wise` and `tensor-wise`. Better to keep this?

Reply:
OK